src/components/contextMenu/AposContextMenu.vue

```diff
@@ -105,26 +111,25 @@ export default {
     },
     buttonState() {
       return this.open ? ['active'] : null;
+    },
+    localCssVar() {
+      // Set the CSS custom property for this specific context menu.
+      return { '--tip-width': `${this.tipWidth}px` };
```

Comment:
At the moment, this is always the same everywhere. We could make this a global variable nice an easy, though that doesn't help with the need to have the value handy for passing into the tip component. ... Or we go with magic numbers in the tip component and here (see line 202) and don't worry about the connection.